sdk/template/template/samples/v1/javascript/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "azure-template-samples-js",
   "private": true,
   "version": "0.1.0",
-  "description": "Azure template client library samples for JavaScript",
+  "description": "Azure Template client library samples for JavaScript",
   "engine": {
     "node": ">=8.0.0"
```

Comment:
README says Node 12, this says Node 8

Reply:
Made these consistent. We're going with Node 12+ for samples (to allow ourselves to use new, good syntax, and Node 10 is going EOL soon anyway).